src/nvim/ex_docmd.c

```diff
@@ -5890,6 +5921,20 @@ static void do_ucmd(exarg_T *eap)
   else
     cmd = USER_CMD_GA(&curbuf->b_ucmds, eap->useridx);
 
+  if (cmd->uc_argt & EX_LUA_CB) {
+    lua_State* L = executor_enter_lua();
+    nlua_pushref(L, cmd->lua_cb);
+    lua_pushlstring(L, (const char*)eap->arg, STRLEN(eap->arg));
+    lua_newtable(L);
+    if (eap->forceit == 1) {
+      lua_pushliteral(L, "bang");
+      lua_pushboolean(L, eap->forceit == 1);
+      lua_rawset(L, -3);
+    }
+    lua_pcall(L, 2, 0, 0);
```

Comment:
I think the `nlua_error" pattern can be used here:

    if (lua_pcall(...)) {
      nlua_error(L, "E5108: Error executing lua: %.*s");
    }